examples/pxScene2d/src/pxWayland.cpp

```diff
@@ -88,6 +88,7 @@ pxWayland::~pxWayland()
 {
 #ifdef ENABLE_PX_WAYLAND_RPC
   rtRemoteUnregisterDisconnectedCallback(pxWayland::remoteDisconnectedCB, this);
+  rtRemoteShutdown();
```

Comment:
Jake, can you please comment on if it's okay to call rtRemoteShutdown() after destroying each pxWayland object?